src/Uno.Toolkit.UI/Behaviors/InputExtensions.cs

```diff
@@ -177,5 +239,37 @@ private static void OnUIElementKeyUp(object sender, KeyRoutedEventArgs e)
 				_ => default,
 			};
 		}
+
+#if __ANDROID__
+		private static ImeAction GetImeActionFromReturnType(ReturnType returnType)
+		{
+			switch (returnType)
+			{
+				case ReturnType.Next: return ImeAction.Next;
+				case ReturnType.Go: return ImeAction.Go;
+				case ReturnType.Search: return ImeAction.Search;
+				case ReturnType.Send: return ImeAction.Send;
+				case ReturnType.Done: return ImeAction.Done;
+				case ReturnType.Default:
+				default: return ImeAction.Unspecified;
+			}
```

Comment:
Can be converted to a [switch expression](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/switch-expression)